<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UltraNerfInferenceController</class>
    <widget class="QWidget" name="UltraNerf_Inference_Controller">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>186</width>
                <height>99</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>UltraNerf Inference</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
                <number>9</number>
            </property>
            <property name="topMargin">
                <number>9</number>
            </property>
            <property name="rightMargin">
                <number>9</number>
            </property>
            <property name="bottomMargin">
                <number>9</number>
            </property>
            <item row="1" column="0">
                <widget class="QLabel" name="labelTop">
                    <property name="text">
                        <string>Upper Point</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="0">
                <widget class="QLabel" name="labelBot">
                    <property name="text">
                        <string>Lower Point</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="1">
                <widget class="QLabel" name="labelXTop">
                    <property name="text">
                        <string>x:</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="3">
                <widget class="QLabel" name="labelYTop">
                    <property name="text">
                        <string>y:</string>
                    </property>
                </widget>
            </item>
            <item row="1" column="5">
                <widget class="QLabel" name="labelZTop">
                    <property name="text">
                        <string>z:</string>
                    </property>
                </widget>
            </item>

            <item row="2" column="1">
                <widget class="QLabel" name="labelXBot">
                    <property name="text">
                        <string>x:</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="3">
                <widget class="QLabel" name="labelYBot">
                    <property name="text">
                        <string>y:</string>
                    </property>
                </widget>
            </item>
            <item row="2" column="5">
                <widget class="QLabel" name="labelZBot">
                    <property name="text">
                        <string>z:</string>
                    </property>
                </widget>
            </item>

            <item row="1" column="2">
                <widget class="QLineEdit" name="xCoorTop">
                </widget>
            </item>
            <item row="1" column="4">
                <widget class="QLineEdit" name="yCoorTop">
                </widget>
            </item>
            <item row="1" column="6">
                <widget class="QLineEdit" name="zCoorTop">
                </widget>
            </item>

            <item row="2" column="2">
                <widget class="QLineEdit" name="xCoorBot">
                </widget>
            </item>
            <item row="2" column="4">
                <widget class="QLineEdit" name="yCoorBot">
                </widget>
            </item>
            <item row="2" column="6">
                <widget class="QLineEdit" name="zCoorBot">
                </widget>
            </item>

            <item row="3" column="0" colspan="3">
                <widget class="QLabel" name="modelPathLabel">
                    <property name="text">
                        <string>Model Path</string>
                    </property>
                </widget>
            </item>

            <item row="3" column="3" colspan="4">
                <widget class="QLineEdit" name="modelPath">
                </widget>
            </item>
            <item row="4" column="0" colspan="4">
                <widget class="QLabel" name="originLabel">
                    <property name="text">
                        <string>Origin of BLines</string>
                    </property>
                </widget>
            </item>
            <item row="4" column="3" colspan="3">
                <widget class="QLineEdit" name="originEnum">
                </widget>
            </item>
            <item row="5" column="0" colspan="3">
                <widget class="QPushButton" name="pushButtonLoadModel">
                    <property name="text">
                        <string>Load Model</string>
                    </property>
                </widget>
            </item>
            <item row="5" column="3" colspan="4">
                <widget class="QPushButton" name="pushButtonRunModel">
                    <property name="text">
                        <string>Create View</string>
                    </property>
                </widget>
            </item>
            <item row="6" column="3" colspan="4">
                <widget class="QPushButton" name="pushButtonGetPoints">
                    <property name="text">
                        <string>Get Points</string>
                    </property>
                </widget>
            </item>
            <item row="0" column="0" colspan="7">
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>This algorithm creates views on a pretrained UltraNeRF model</string>
                    </property>
                    <property name="wordWrap">
                        <bool>true</bool>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>